secator/providers/vulners.py

```
@@ -0,0 +1,61 @@
+		"""
+		api_key = CONFIG.addons.vulners.api_key
+		enabled = CONFIG.addons.vulners.enabled
+		if not enabled:
+			return None
+		if not api_key:
+			raise ValueError('Vulners API key not set. Please run secator config set addons.vulners.api_key <API_KEY>')
+		try:
+			import vulners
+			vulners_client = vulners.VulnersApi(api_key=api_key)
+			cve_data = vulners_client.search.get_bulletin(cve_id, fields=["*"])
+			return cve_data
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)